ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -574,6 +625,40 @@ export default {
         },
       },
     },
+    {
+      id: 'E9S Summon',
+      netRegex: NetRegexes.ability({ id: '5019', source: 'Cloud Of Darkness', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5019', source: 'Wolke Der Dunkelheit', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5019', source: 'Nuage De Ténèbres', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5019', source: '暗闇の雲', capture: false }),
+      netRegexCn: NetRegexes.ability({ id: '5019', source: '暗黑之云', capture: false }),
+      netRegexKo: NetRegexes.ability({ id: '5019', source: '어둠의 구름', capture: false }),
+      promise: async (data) => {
+        const cloneOfDarknessLocaleNames = {
+          en: 'Clone Of Darkness',
```

Comment:
`de: 'Klon der Dunkelheit',`
`fr: 'Nuage de Ténèbres',`
Hi, if possible to add this.

Reply:
ja: '幻影の雲',

Reply:
`cn: '幻影之云',` 😊

Reply:
@VanSlanzar is this right? `Nuage de Ténèbres` for both the boss and clones?

Reply:
i think it should be `Nuée de Ténèbres` for the clones as the cloud itself is `Nuage de Ténèbres`

Reply:
In xivapi it's `nuage de ténèbres`.  But we can to Writte `Clone de Ténèbres `. It's better.

Reply:
this is the mentioned clone of darkness in xivapi: https://xivapi.com/BNpcName/9765

Reply:
Yes this is what I saw on xivapi, but I only see symbols. `Name_fr":"nu\u00e9e de T\u00e9n\u00e8bres`.
Is there a way to see the translation correctly?
Otherwise I just checked my logs because I just started raiding last night, Nuée de Ténèbres is correct.
Sorry for this confusion.

Reply:
i have either a random json plugin for my browser installed or use postman for requests. Both show it as `Nuée de Ténèbres`

Reply:
you can link your plugin ty.

Reply:
Displays fine in Firefox.
![image](https://user-images.githubusercontent.com/6119598/102650021-351d0380-4138-11eb-9890-0eb391d1b690.png)

Reply:
I have taked jsonview on chrome, ty for your help. that will help me